### PR TITLE
👁  On-boarding top bar

### PR DESCRIPTION
**Pull request by stepien4soft**
Closes: #1562

### CODE REVIEW

**Comment by DevNikita16** [comment]
👀

File: packages/ui/src/app/components/OnboardingOverlay/OnBoardingOverlay.tsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+  const onBoardingSteps = asOnBoardingSteps(steps, status)
+
+  return (
+    <MainWrapper>
+      <Wrapper>
+        <TextContainer>
+          <TextHuge bold>Become a member</TextHuge>
+          <TextSmall onClick={() => setIsOpen((prev) => !prev)}>
+            Show how <ArrowDownExpandedIcon /> {/*todo add icons and make them rotate on open*/}
```
Comment (DevNikita16) [inline packages/ui/src/app/components/OnboardingOverlay/OnBoardingOverlay.tsx:86]: Probably should be handled in this PR :D

File: packages/ui/test/app/pages/components/OnBoardingOverlay.test.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      expect(getByText('How to become a member?')).toBeDefined()
+    })
+
+    it('Install plugin', async () => {
+      useMyAccounts.error = 'EXTENSION'
+
+      const { getByText } = await renderComponent()
+      const pluginCircle = getByText('Add Polkadot plugin')?.parentElement?.previousElementSibling
+
+      expect(pluginCircle).toHaveStyle(`background-color: ${Colors.Blue[500]}`)
```
Comment (DevNikita16) [inline packages/ui/test/app/pages/components/OnBoardingOverlay.test.tsx:74]: 👍👍

**Review by DevNikita16 — CHANGES_REQUESTED** [review]
Overall looks fine! 🚀 Just handle that icon

File: packages/ui/src/app/components/OnboardingOverlay/OnBoardingOverlay.tsx [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    if (index === activeIndex) {
+      return { ...step, type: 'active' }
+    }
+    return { ...step, type: 'next' }
+  })
+}
+
+export const OnBoardingOverlay = () => {
+  const { isLoading, status } = useOnBoardingStatus()
+  const [isOpen, setIsOpen] = useState(false)
```
Comment (DevNikita16) [inline packages/ui/src/app/components/OnboardingOverlay/OnBoardingOverlay.tsx:72]: Here we should use `useToggle` hook

Comment (WRadoslaw) [reply packages/ui/src/app/components/OnboardingOverlay/OnBoardingOverlay.tsx]: Fixed

File: packages/ui/src/app/components/OnboardingOverlay/OnboardingOverlay.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+export default {
+  title: 'App/OnboardingOverlay',
+  component: OnBoardingOverlay,
+} as Meta
+
+const Template: Story = (args) => (
+  <TemplateBlock>
+    <OnBoardingOverlay {...args} />
+  </TemplateBlock>
+)
```
Comment (jodator) [inline packages/ui/src/app/components/OnboardingOverlay/OnboardingOverlay.stories.tsx:16]: I tried to test this on the storybook and, at least for me, it wasn't rendering anything. I would add some controls here to simulate the change in the onboarding status. This way we could test or change the UI quickly.

**Review by jodator — CHANGES_REQUESTED** [review]
The component works very well :) I had some trouble with testing it, but after reading the requirements it was easy :D 

However, I wasn't able to test it using storybook and I think that would be nice to have it. This way we could check or change the UI quicker. I think that it should be doable to add a control to change the onboarding status and simulate the required state for the component using contexts. WDYT?

File: packages/ui/src/app/components/OnboardingOverlay/OnboardingOverlay.stories.tsx (hunk 4 above)
Comment (WRadoslaw) [reply packages/ui/src/app/components/OnboardingOverlay/OnboardingOverlay.stories.tsx]: Yeah, I forgot to update story after hook integration, I've take care of it and it should be working now :)

**Review by DevNikita16 — APPROVED** [review]

**Pull request merged**